tox.ini

```diff
@@ -135,8 +135,10 @@ commands =
 basepython: python3.8
 recreate = True
 deps =
-  -e {toxinidir}/opentelemetry-python-contrib/instrumentation/opentelemetry-instrumentation-grpc
+  -e {toxinidir}/opentelemetry-api
```

Comment:
I was about to tell you that I still can't run when I saw these changes. Probably not corresponding to this issue, but it smells like something is wrong when we have to install the GRPC instrumentation to run lint.

Reply:
Yes I agree with you, but I didn't look further into why they fail without this package :/

Reply:
@ocelotl I fixed the issue with grpc requirement! By making it as a requirement to the Example App, it should have been but it was not a dependency of that app.

@mariojonke has a great issue that helped me figure out my problem here! https://github.com/open-telemetry/opentelemetry-python/issues/1391